akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -472,9 +468,108 @@ object Source {
    *
    * @see [[Source.lazily]]
    */
+  @deprecated("Use 'Source.lazyFuture' instead", "2.6.0")
   def lazilyAsync[T](create: () => Future[T]): Source[T, Future[NotUsed]] =
```

Comment:
you said that my table was wrong for this one

> Source.lazilyAsync becomes lazyFutureSource

but I think it's lazyFuture

Reply:
Oh, sorry my bad (I blame it on the inconsistent old naming. ;) )